jmclient/jmclient/taker_utils.py

```diff
@@ -77,6 +78,7 @@ def direct_send(wallet_service, amount, mixdepth, destination, answeryes=False,
 
     txtype = wallet_service.get_txtype()
     if amount == 0:
+        #doing a sweep
```

Comment:
Perhaps add an `assert custom_change_address is None` or similar to this sweep section?

Reply:
Good idea. But I'll add it to the sanity checking section above to keep things organized.